classes/local/factor/object_factor.php

```diff
@@ -36,6 +36,7 @@ public function define_add_factor_form_definition_after_data($mform);
     public function define_login_form($mform);
     public function validation($data);
     public function add_user_factor($data);
-    public function get_user_factors($user);
+    public function get_all_user_factors($user);
```

Comment:
Can you please add php docs for all of these as you go so it's clear the intent of each of the methods. eg get_all_user_factors() isn't obvious to me what it's doing (yet)

Reply:
Will add php docs in further PR